app/src/main/java/io/neurolab/main/output/visual/SpaceAnimationVisuals.java

```diff
@@ -70,14 +69,83 @@ public void animateRocket(double[] data, Activity activity) {
 
         double value = Double.parseDouble(Double.toString(data[count]).substring(0, 3));
         valueAnimator[0] = ValueAnimator.ofFloat(0f, (float) -value);
-        valueAnimator[0].setInterpolator(new LinearInterpolator());
-        valueAnimator[0].setRepeatMode(ValueAnimator.REVERSE);
-        valueAnimator[0].setRepeatCount(3);
-        valueAnimator[0].setDuration(1000);
+        valueAnimator[0].setInterpolator(new AccelerateInterpolator());
+        valueAnimator[0].setDuration(500);
```

Comment:
If this is responsible for the up animation, then we can make it `900`. Looks less abrupt.